plugins/BeamerPresentation/plugin.ini

```diff
@@ -0,0 +1,15 @@
+[about]
+## Author / Copyright notice
+author=Lukas Heindl
+
+description=Step through LaTeX beamer presentation
```

Comment:
Maybe add a bit more info on how it works.

Reply:
True. I always feel the way we're presenting this setting in the plugin manager (only one line) this should be short. Also to not clutter the overview page of the plugin manager (maybe adding a new setting (something like `usage`) which is only shown when clicking on the plugin in the plugin manager would solve this, but that's not for this PR).
I'll make the description a bit longer (probably explaining that this plugin uses one page per slide...)

Reply:
Should be addressed.